enterprise/backend/src/metabase_enterprise/sso/integrations/sso_settings.clj

```diff
@@ -255,3 +256,12 @@ on your IdP, this usually looks something like http://www.example.com/141xkex604
   :visibility :public
   :setter     :none
   :getter     (fn [] (or (saml-enabled) (jwt-enabled))))
+
+(defenterprise sso-info
```

Comment:
nit - maybe make this function name more SAML-specific since it's not exposing other SSO settings
```suggestion
(defenterprise saml-settings
```

Reply:
I'll remove it since we can use sso-settings directly in the ee code